source/bringauto/external_client/connection/communication/MqttCommunication.cpp

```diff
@@ -64,7 +64,7 @@ void MqttCommunication::initializeConnection() {
 	if (client_ != nullptr && client_->is_connected()) {
 		return;
 	}
-	else if (client_ != nullptr) {
+	if (client_ != nullptr) {
```

Comment:
Why was the "else" deleted? Oo, I see if it is connected I want to close the connection in this case.